src/platform/site-wide/sass/merger.scss

```diff
@@ -815,3 +810,9 @@ img[data-srcset] {
 .profile-nav {
   margin-top: 6px;
 }
+
+.usa-alert {
+  .top-of-page {
+    margin-top: 0;
+  }
+}
```

Comment:
Maybe this would be better in the design system? Not sure, tiny tweaks like this generally make sense in a file like this.